posts/services/feed.py

```
@@ -48,6 +48,7 @@ def get_posts_feed(
     show_on_homepage: bool = None,
     following: bool = None,
     upvoted_by: int = None,
+    commented_by: int = None,
```

Ruff (0.15.1)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)